### DISCUSSION TITLE
How to save only once

### DISCUSSION THREAD

**Discussion by BCRoaster** [artisan-roaster-scope/artisan#1716] [General]
After ending a roast with drop, I stop the recording to prevent the long tail after drop. At that point it asks if I want to save the file. I DO want to save but after I have cooled the beans and entered the roasted weight. As it stands I save the file, click OFF, then File/open recent to enter roasted weight. Then I save the file and it shows the % weight loss in stats and the saved file name, but then I have to delete the saved file that shows weight loss 100%. How do I save with actual weight loss and not have to delete the earlier saved file? Sorry if this has already been discussed.

**Comment by poundy** [comment]
I wonder if your process you've outlined is actually not one that anyone uses, as it's too "slow" between batches - you need to completely cool your roast, then weigh it, then do data entry before you can move on to the next batch.  The workflow I use is to drop the current batch, get the machine reconfigured for the preheat/BBP I use (gas to full, airflow to my idle setting) and get the next weighed green coffee ready to load in the hopper. Then once my environment is ready to charge I'll start Artisan recording, load to the hopper, and charge once I hit my numbers.  Then, I'll worry about my roasted batch, weighing it and then recording the roast weight using Artisan Viewer which automatically opens the last roasted batch.

**Comment by roasterdave** [comment]
>How do I save with actual weight loss and not have to delete the earlier saved file?

Artisan cannot rename a file.  Maybe a future enhancement candidate.  For now you will need to delete the earlier file.

**Reply by BCRoaster** [reply]
Thanks Dave. It's just a small hassle, but one I do often. I can live with it. I'm a home roaster, so my BBP is somewhat relaxed compared to someone roasting commercially.

**Reply by poundy** [reply]
I'd still suggest you look at running the second instance of Artisan (which runs as Artisan Viewer) and you can avoid that.
